File: xml-performance/test-drive-xml-jaxb/src/main/java/org/jesperancinha/test/drive/jaxb/books/xml/Books.java
package org.jesperancinha.test.drive.jaxb.books.xml;

import jakarta.xml.bind.annotation.XmlAccessType;
import jakarta.xml.bind.annotation.XmlAccessorType;
import jakarta.xml.bind.annotation.XmlElement;
import jakarta.xml.bind.annotation.XmlRootElement;

import java.util.List;

@XmlAccessorType(XmlAccessType.FIELD)
@XmlRootElement(name = "books",
        namespace = "")
public class Books {

    @XmlElement(name = "book")
    public List<Book> book;

    public List<Book> getBooks() {
        return book;
    }
}

File: jeorg-kotlin-apps/performance-parsers/xml-performance/test-drive-xml-jaxb/src/main/kotlin/org/jesperancinha/test/drive/jaxb/books/xml/Books.kt
package org.jesperancinha.test.drive.jaxb.books.xml

import jakarta.xml.bind.annotation.XmlAccessType
import jakarta.xml.bind.annotation.XmlAccessorType
import jakarta.xml.bind.annotation.XmlElement
import jakarta.xml.bind.annotation.XmlRootElement

@XmlAccessorType(XmlAccessType.FIELD)
@XmlRootElement(name = "books", namespace = "")
class Books {
    @XmlElement(name = "book", namespace = "http://www.joaofilipesabinoesperancinha.nl")
    lateinit var books: List<Book>
}